### DISCUSSION TITLE
Ignore mode with MacOS

### DISCUSSION THREAD

**Discussion by hansbogert** [tridactyl/tridactyl#4770] [General]
I'm cursed with a MacOS based laptop at work. To my knowledge there's no 'insert' key that actually works like one on a mac. the `Fn+ENTER` does not seem to work reliably.

How do other people do `shift-insert` to activate ignore mode?

If this is just 'broken' on MacOS, how do you change the default ignore shortcut to something else? The alternative `ctrl-alt-esc` is really unfriendly to the fingers so it's not really an option imo.

**Comment by bovine3dom** [comment]
You have a few options:

```
        "<S-Insert>": "mode ignore",
        "<AC-Escape>": "mode ignore",
        "<AC-`>": "mode ignore",
        "<S-Escape>": "mode ignore",
```

Does shift-escape not work?

You can use `:bindwizard`, press the keys you'd like to bind to, then `<Enter>` to help you make a bind. Then just `:bind [keys] mode ignore` and `:bind --mode=ignore [keys] mode normal` to make a toggle

**Reply by hansbogert** [reply]
I was unaware of `<S-Escape>`, very workable alternative.

Also thank you for the other instructions!
